CSS/styles.css

```diff
@@ -0,0 +1,123 @@
+nav, h1 {
+
+  text-align: center;
+
+}
+
```

Comment:
Please remove unneeded line breaks in the CSS rules.  🔨 

should be:
`selector {`
`  rule: value;`
`}`